modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_claim_service_base.rb

```diff
@@ -44,6 +46,33 @@ def get_error_message(error)
         end
       end
 
+      def get_error_key(error_message)
+        error_message.dig(:messages, 0, :key) || error_message
+      end
+
+      def get_error_text(error_message)
+        error_message.dig(:messages, 0, :text) || error_message
+      end
+
+      def get_error_status_code(error)
+        if error.respond_to? :status_code
+          error.status_code
+        else
+          "No status code for error: #{error}"
+        end
+      end
+
+      def will_retry?(error)
+        msg = if error.respond_to? :original_body
+                error&.original_body&.[](:messages)&.[](0)&.[](:key)
```

Comment:
Shouldn't we use the `get_error_key()` from above?

Reply:
yes, that is a good catch